content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/1-mysql.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "MySQL"
+metaTitle: "Palette Dev Engine MySQL Service"
+metaDescription: "Palette Dev Engine MySQL Service"
+hideToC: false
+type: "appTier"
+category: ['databases']
+fullWidth: false
+logoUrl: "https://registry.dev.spectrocloud.com/v1/mysql-operator/blobs/sha256:2d59bc428916752528280eac03330d712164163e2f3c476409f5c25d8a7c2778?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# MySQL
+
+[MySQL](https://www.mysql.com/) is an open-source relational database management system commonly used in web applications and other software that requires a database. It is known for its reliability, ease of use, and flexibility. MySQL is covered under the GNU license and uses structured query language (SQL) to manage data with the following properties:
+
+<br />
+
+* Creates a database for storing and manipulating data and defining the relationship of each table.
+
+
+* Clients can retrieve and manipulate data by creating SQL queries and submitting them to the MySQL instance.
+
+
+# Add MySQL to App Profile
+
+Use the following steps to add MySQL to an app profile.
+
+<br />
+
+## Prerequisite
+
+A Spectro Cloud [account](https://www.spectrocloud.com/get-started/).
+
+<br />
+
+## Enablement
+
+1. Log in to [Palette](console.spectrocloud.com).
+
+
+2. On the right side of the window, click on the **User Menu**. Once the user menu is expanded, click on **Switch to App Mode**.
```

Comment:
```suggestion
2. On the right side of the window, click on the **User Menu**, then select **Switch to App Mode**.
```

Reply:
I'd suggest this wording throughout so we don't have so many "Click on" mentions.